spec/schemas/_core.termvectors.yaml

```diff
@@ -11,21 +11,21 @@ components:
       properties:
         max_doc_freq:
           description: |-
-            Ignore words which occur in more than this many docs.
-            Defaults to unbounded.
+            Ignore words which occur in more than the number of docs indicated.
+            Default is unbounded.
           type: number
         max_num_terms:
-          description: Maximum number of terms that must be returned per field.
+          description: The maximum number of terms that must be returned per field.
```

Comment:
This is redundant. Either "The number of terms that must be returned per field" or "The maximum number of terms that should be returned per field"